test/util/scripts/env.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
-export KAFKA_ID=kafka
+export KAFKA_ID=cl_kafka
```

Comment:
kafka is not specific for central-ledger ml-api-adapter also uses the same instance